src/objc/properties.rs

```diff
@@ -62,3 +63,32 @@ pub(super) fn objc_setProperty(
         release(env, old);
     }
 }
+
+// note: https://opensource.apple.com/source/objc4/objc4-723/runtime/objc-accessors.mm.auto.html
+//       says that hasStrong is unused, but other versions of the source seem to use it. Not sure what to do here.
+pub(super) fn objc_copyStruct(
+    env: &mut Environment,
+    dest: MutVoidPtr,
+    src: ConstVoidPtr,
+    size: GuestUSize,
+    _atomic: bool,
+    _hasStrong: bool,
+) {
+    // TODO: implement atomic and hasStrong support
+    // Right now this is a clone of memmove.
```

Comment:
You could actually just call `crate::libc::string::memmove` if you want to.

Reply:
I wasn't sure if it made sense to call string's memmove. Maybe the memmove function should be moved to the top-level libc module?

Reply:
This reminded me that Iwas already thinking that I should just use Rust's built-in `memmove` equivalent, rather than writing my own, so I did that now: bbf6c68d820d75849febd1747c4881006bd9f06f.

That's not a `libc` function so you can use it here, perhaps?

Reply:
Yep. I'm having trouble rebasing onto latest trunk, though, so would you be able to make that change?